### DISCUSSION TITLE
[v0.1300.0.3] Pressure shouldn't apply to diving suits

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#5321] [Unstable feedback]
Unstable introduced two new diving suits, Abyss and Combat, as well as difference pressure resistances for the suits

All three suits have different pressure resistances: Combat < Normal < Abyss

Unfortunately, the combat suit breaks from pressure as early as 2500m, which is a depth reached in the second biome of the game
The normal suits will explode as early as the apothic plateau
**The brand new combat suit, while powerful, becomes completely useless after the first biome, while the abyss suit is mandatory to progress.** 

While the abyss suit can be seen as an extension of the forced _"upgrade hull to resist pressure"_ system (not a fan but it makes sense) , combat suits are made obsolete very early in the campaign.

Pressure shouldn't be a factor in suit balance, otherwise 2 out of the 3 suits will inevitably be thrown in the recycler.

Oxygen consumption, crafting costs, speed, field of view, even running the suits on batteries should be the balancing factors, not pressure which will always be increasing and always break the least pressure resistant suits as you advance in the campaign.

**Comment by TheNinjaScout** [comment]
I agree with everything said above, though I would appreciate it if the pressure resistance between the suits was kept with a minimal (practically neglectable) difference between them for modding purposes.
(The way the suits are set up at the moment allows modders to check the current pressure resistance of a character, and based on what that number is, it's possible to tell which suit the character is wearing, which can be rather useful)
